src/core/Akka/IO/TcpSettings.cs

```diff
@@ -44,7 +48,9 @@ public static TcpSettings Create(Config config)
                 bufferPoolConfigPath: config.GetString("buffer-pool", "akka.io.tcp.disabled-buffer-pool"),
                 initialSocketAsyncEventArgs: config.GetInt("nr-of-socket-async-event-args", 32),
                 traceLogging: config.GetBoolean("trace-logging", false),
-                batchAcceptLimit: config.GetInt("batch-accept-limit", 10),
+                batchAcceptLimit: config.GetString("batch-accept-limit") == "scale-to-cpus"
```

Comment:
Handle the new `scale-to-cpus` value we accept for `akka.io.tcp.batch-accept-limit`